src/EmailAddress.php

```diff
@@ -95,6 +104,16 @@ public function equals($other, bool $caseInsensitive = false)
         }
     }
 
+    /**
+     * Returns true if the email contains one of our defined disallowed aliases.
+     *
+     * @return bool
+     */
+    public function hasDisallowedAlias(): bool
```

Comment:
These aren't inherently disallowed email addresses. They're just well known email aliases. It looks like there's even an [RFC](https://tools.ietf.org/html/rfc2142) listing them. Maybe rename this to `isStandardRoleAlias()`?